database/things/iterm2.pldb

```diff
@@ -0,0 +1,24 @@
+title Iterm2
+type application
+description iTerm2 is a replacement for Terminal and the successor to iTerm. It works on Macs with macOS 10.14 or newer. iTerm2 brings the terminal into the modern age with features you never knew you always wanted.
+website hhttps://iterm2.com/
+documentation https://iterm2.com/documentation.html
+originCommunity https://github.com/gnachman/iTerm2
+domainName iterm2.com
+
+writtenIn c python
+
+githubRepo https://github.com/gnachman/iTerm2
+ stars 12900
+ forks 1100
+ updated 2022
+ firstCommit 1996
+ description iTerm2 is a terminal emulator for Mac OS X that does amazing things.
+
+wikipedia https://en.wikipedia.org/wiki/ITerm2
+ related git magit sourcetree
+ pageId 44296721
+ dailyPageViews 84
+
+releaseNotesUrl https://iterm2.com/news.html
+officialBlogUrl https://iterm2.com/news.html
```

Comment:
Since there is no real blog on the site, that I can see, might it be best to not have this URL?